pkg/controller/appdefinition/endpoints.go

```diff
@@ -68,7 +68,7 @@ func serviceEndpoints(req router.Request, app *v1.AppInstance) (endpoints []v1.E
 				endpoints = append(endpoints, v1.Endpoint{
 					Target:     containerName,
 					TargetPort: port.TargetPort.IntVal,
-					Address:    fmt.Sprintf("<Pending Ingress>:%d", port.Port),
+					Address:    fmt.Sprintf("<Pending>:%d", port.Port),
```

Comment:
This is actually slightly more complicated...there are two ways that end points are exposed:
1. If the port being published is an HTTP port, ingress is used
2. If the port being published is something else, like a TCP port, a service load balancer is used

I'd like this message to reflect that. I think the port should have the protocol in there somewhere.

Reply:
```
jsilverio@joshuas-mbp acorn % ./bin/acorn run -p 9000 tcp
cool-violet
STATUS: ENDPOINTS[] HEALTHY[] UPTODATE[] 
STATUS: ENDPOINTS[] HEALTHY[0] UPTODATE[0] pending
STATUS: ENDPOINTS[] HEALTHY[0/1] UPTODATE[1] [containers: echo ContainerCreating]
STATUS: ENDPOINTS[tcp://<Pending> => echo:9000] HEALTHY[0/1] UPTODATE[1] [containers: echo ContainerCreating]
STATUS: ENDPOINTS[tcp://127.0.0.1:9000 => echo:9000] HEALTHY[0/1] UPTODATE[1] [containers: echo ContainerCreating]

┌────────────────────────────────────────────────────────────────────────────────┐
| STATUS: ENDPOINTS[tcp://127.0.0.1:9000 => echo:9000] HEALTHY[1] UPTODATE[1] OK |
└────────────────────────────────────────────────────────────────────────────────┘
jsilverio@joshuas-mbp acorn % ./bin/acorn                
jsilverio@joshuas-mbp acorn % acorn run -P ghcr.io/acorn-io/library/hello-world

cold-fog
STATUS: ENDPOINTS[] HEALTHY[] UPTODATE[] 
STATUS: ENDPOINTS[] HEALTHY[0] UPTODATE[0] pending
STATUS: ENDPOINTS[] HEALTHY[0/1] UPTODATE[1] [containers: webapp ContainerCreating]
STATUS: ENDPOINTS[http://<Pending Ingress> => webapp:80] HEALTHY[0/1] UPTODATE[1] [containers: webapp ContainerCreating]
STATUS: ENDPOINTS[http://<Pending Ingress> => webapp:80] HEALTHY[0/1] UPTODATE[1] [containers: webapp is not ready]
┌────────────────────────────────────────────────────────────────────────────────────┐
| STATUS: ENDPOINTS[http://<Pending Ingress> => webapp:80] HEALTHY[1] UPTODATE[1] OK |
```

Is this more the functionality you were expecting? @cjellick